test/e2e/collect-ci-artifacts.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+: "${KUBECONFIG:?}"
+: "${ARTIFACT_DIR:?}"
+: "${KUBECTL:=oc}"
+
+function ensure_kubectl() {
+    # Check whether we're running in downstream CI environments as the location for the
+    # "oc" binary is located at "/cli/oc" path. This is problematic as the /cli directory
+    # doesn't exist in the $PATH environment variable, which causes issues when running
+    # this script via the exec.Command Golang function.
+    if [[ "$OPENSHIFT_CI" == "true" ]]; then
```

Comment:
Note: We could also perform a `-d` check on whether the /cli directory exists. The OPENSHIFT_CI variable will always be set when running the e2e suite in downstream CI environments, so it feels like we can make a really solid assumption on where the oc binary will live going forward.

This isn't completely future proof in the case this directory location changes, but we should be fine given we're stat(ing) the configured kubectl binary immediately after performing this conditional check. In the future, updating this script and adding a conditional check for validating this directory exists could produce a more robust implementation.